phabricator.py

```diff
@@ -37,8 +39,25 @@ def run(self):
         # Find the preselected branch
         git_branch = settings.get('branch')
 
-        # If no preselected branch is provided
-        if git_branch == None:
+        if git_branch is None and settings.get('use_arc_land_onto_default', False):
```

Comment:
IMO that feels more brittle since `branch` might be an instance of a dict or string. And in the former case we have to check for presence of key in dict and then check the value at said key.
It feels like a strange case of overloading to me.

Most plugins just add more settings as need be. I do think that it makes it a little difficult to explain the behavior when both the 'branch' and 'use_arc_land_onto_default' keys are set, but it makes the code easier to grok.

Eventually happy to do whatever you guys think is the best.